src/views/Resources.vue

```diff
@@ -3,51 +3,98 @@
     <v-row justify="center">
       <v-col cols="10" max-width="1600">
         <v-card class="elevation-12">
-          <div class="text-center">
-            <h1>Resources table</h1>
-          </div>
-          <div class="d-flex justify-end">
-            <v-col cols="12" sm="6" md="4" class="text-end">
-              <v-btn rounded="xs" size="x-large" color="red" to="/resources/add"
-                >Add resource</v-btn
-              >
-            </v-col>
+          <v-container class="text-center text-h4 font-weight-bold">
+            {{
+              sortChoice === "All"
+                ? "All resources table"
+                : `${sortChoice}'s resources table`
+            }}
+          </v-container>
+          <div class="d-flex justify-space-between">
+            <v-menu>
+              <template v-slot:activator="{ props }">
+                <v-btn
+                  variant="outlined"
+                  class="mx-4"
+                  rounded="xs"
+                  :size="isSmAndDown ? 'small' : 'x-large'"
+                  color="red"
+                  v-bind="props"
+                >
+                  Sorting
+                </v-btn>
+              </template>
+              <v-list>
+                <v-list-item
+                  v-for="(item, index) in items"
+                  :key="index"
+                  :value="index"
+                >
+                  <v-list-item-title @click="clickHandler(item.title)">{{
+                    item.title
+                  }}</v-list-item-title>
+                </v-list-item>
```

Comment:
After applying suggestion below, this becomes:
```suggestion
                 <v-list-item
                  v-for="resourceType in resourceTypes"
                  :key="resourceType"
                  :value="resourceType"
                >
                  <v-list-item-title @click="filterResourcesByType(resourceType)">{{
                    resourceType
                  }}</v-list-item-title>
```